pyproject.toml

```diff
@@ -26,7 +26,7 @@ hydra-core = "^1.3.2"
 pyliftover = "^0.4"
 numpy = "^1.26.2"
 hail = "0.2.127"
-wandb = ">=0.16.2,<0.19.0"
+wandb = ">=0.16.2,<0.18.0"
```

Comment:
is it on purpose?

Reply:
sorry I made a mess with the git history. Reverting before the merge